setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+with open("requirements.txt") as f:
+    requirements = f.read().splitlines()
+
+
+setup(
+    name="smtb",
+    version="0.1.0",
+    author="Ilya Senatorov",
+    author_email="il.senatorov@protonmail.com",
+    url="https://github.com/kalininalab/SMTB2024",
+    packages=["smtb"],
+    install_requires=requirements,
+    python_requires=">=3.11",
```

Comment:
Again is 3.11 maybe too high, maybe 3.10?

Reply:
agreed

Reply:
I don't think, 3.11 is too high. I have another project with similar dependencies that is run on 3.11 and it works fine, so, I wouldn't expect any problems. But we can downgrade to 3.10 if you want.

Reply:
Yes, maybe `>=3.10`